Makefile

```diff
@@ -48,8 +42,6 @@ _IMAGE = $(_DOCKER_REGISTRY)/$(_TEST_PROJECT)/$(DEV_IMAGE_NAME)
 _TESTRUNNER_PORT = 80
 endif
 
-_DEV_IMAGE_STREAM = $(DEV_IMAGE_NAME):latest
```

Comment:
The other changes look good but this removing of _DEV_IMAGE_STREAM and using just _IMAGE will cause trouble when testing a development image in Online OpenShift before it's released to registry.access.redhat.com. When testing we're pushing the image to the internal repository because the docker-registry.engineering.redhat.com is not reachable. And then we use the resulting image stream from the internal repository in the tests. Did you try running tests with a dev image? I believe it should fail with the changes

Reply:
What command I should check exactly? `make run-caching-service-remotely`? 

BTW, since this doesn't touch any modules, I'll spin up Cache Service build now and we'll figure this PR out "asynchronously".

Reply:
Yes. Either that or test-remote. This change affects both.

Reply:
I've just ran some tests and you're absolutely right... let me revert that piece.

Reply:
Should be good now.